Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -219,18 +218,20 @@ public void SetLocalContext(ContextDictionary localContext)
     /// <param name="executionLocation"></param>
     public ContextDictionary GetClientContext(ApplicationContext.ExecutionLocations executionLocation)
     {
-      if (ClientContext is null)
+      ContextDictionary clientContext;
+      var sessionManager = ApplicationContext.GetRequiredService<ISessionManager>();
+      var session = sessionManager.GetSession();
+      session.TryGetValue("clientContext", out var result);
```

Comment:
Same suggestion like `GetLocalContext`.

Reply:
Does this increase or decrease readability? I argue that it decreases readability.